### PR TITLE
add DT for aat

### PR DESCRIPTION
**Pull request by naikajah**

### CODE REVIEW

File: k8s/aat/common/monitoring/oneagent.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: helm.fluxcd.io/v1
```
Comment (timja) [inline k8s/aat/common/monitoring/oneagent.yaml:2]: thoughts on moving this to kustomize? since we're at 3 environments now?

Comment (naikajah) [reply k8s/aat/common/monitoring/oneagent.yaml:2]: Yeah, I guess you are right. I will move this to Kustomize.

File: k8s/namespaces/monitoring/kustomization.yaml [hunk 2]
```diff
@@ -2,4 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - prom-operator.yaml
-  - nodelocaldns-alerts-rules.yaml
\ No newline at end of file
+  - nodelocaldns-alerts-rules.yaml
+  - oneagent.yaml
```
Comment (adusumillipraveen) [inline k8s/namespaces/monitoring/kustomization.yaml:6]: this should be s separate kustomization as it's not needed on every cluster.

File: k8s/namespaces/monitoring/dynatrace/patches/perftest/oneagent.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: oneagent-operator
+  namespace: monitoring
+spec:
+  values:
+    oneagent:
+      apiUrl: https://yrk32651.live.dynatrace.com/api
```
Comment (adusumillipraveen) [inline k8s/namespaces/monitoring/dynatrace/patches/perftest/oneagent.yaml:10]: This can move to base if its same

**Pull request closed without merging**